.github/workflows/tests.yml

```
@@ -109,6 +109,9 @@ jobs:
             export COILED_SOFTWARE_NAME=${{ env.COILED_SOFTWARE_NAME }}
           else
             export EXTRA_OPTIONS=" "
+            export RUNTIME_VERSION_FORMATTED=$(echo "${{ matrix.runtime-version }}" | sed 's/\./-/g' )
+            export PYTHON_VERSION_FORMATTED=$(echo "${{ matrix.python-version }}" | sed 's/\.//g' )
+            export DEFAULT_SOFTWARE_NAME=dask-engineering/coiled-runtime-$RUNTIME_VERSION_FORMATTED-py$PYTHON_VERSION_FORMATTED
```

We could just use `COILED_SOFTWARE_NAME` I was wondering if we wanted to be more explicit about the env name, But happy to change it to with the respective change on the conf test
```suggestion
            export COILED_SOFTWARE_NAME=dask-engineering/coiled-runtime-$RUNTIME_VERSION_FORMATTED-py$PYTHON_VERSION_FORMATTED
```